README.md

```diff
@@ -3,3 +3,8 @@
 | Route | Description | Parameters | Expected Output |
 |-------|-------------|------------|-----------------|
 | `POST /api/login`| logs the user in | body: `{ email: string, password: string}` | `{ token: string, refresh_token: string }` |
+| `POST /api/user?=1/create`| Create a list | body `{ title: string, description: string}` | `{ title: string, description: string}` |
+| `POST /api/sign`| Sign up user in | body: `{ firstName: string, lastName: string, email: string, password: string}` | `{ token: string, refresh_token: string }` |
+| `GET /api/user?=1/review`| Review task | body: | `{ title: string, description: string}`|
+| `PUT /api/user?=1/review/edit`| Edit Task | body: `{ title: string, description: string}` | `{ title: string, description: string}` | 
```

Comment:
these paths seems to me that are related to tasks. Task is another entity in our system so it should be something like this (if you are following the REST pattern):
`PATCH /api/task/{id}/review`
`PATCH /api/task/{id}/edit`

Reply:
| `GET /api/user/task` | List all task | body `{title: string, description: string}` | `{title: string, description: string}` |
| `GET /api/user/task/{id}` | List specific task | body `{title: string, description: string}` | `{title: string, description: string}` |
| `PUT /api/user/task/{id}/edit` | Edit specific task | body `{title: string, description: string}` | `{title: string, description: string}` |
| `PUT /api/user/task/{id}/done` | Realized specific task | body `{title: string, description: string}` | `{title: string, description: string}` |
| `DELETE /api/user/task/{id}/delete` | Delete specific task | body `{title: string, description: string}` | `{title: null, description: null}` |